### PR TITLE
Feature/form builder getting started content

### PR DESCRIPTION
**Pull request by jonathanprozzi**
## GitHub Issues

- [Documentation #465](https://github.com/HausDAO/daohaus-monorepo/issues/465)
- [Documentation #473](https://github.com/HausDAO/daohaus-monorepo/issues/473)
- [Documentation #694](https://github.com/HausDAO/daohaus-monorepo/issues/694)

## Overview 

This adds a good bit of content for Form Builder in particular. I now have a good grasp on how to present Form Builder in our docs and have plans to keep iterating and improving but wanted to get these in as they are so far!

@earth2travis

### CODE REVIEW

File: pages/getting-started.mdx [hunk 1]
```diff
@@ -17,36 +17,66 @@ We're using a monorepo to manage our libraries and apps. Each library can be ins
 Our libraries in the libs folder are structured to be consumed by our apps as well as used by external developers using our tooling.
 
 **Utility Libraries**
-The [Common](../sdk/utilities/common-utilities/overview) and [Contract](../sdk/utilities/contract-utilities/overview) libraries provide foundational utility that can be composed and integrated into applications.
+These are libraries that are used to provide foundational utility that can be composed and integrated into applications.
```
Comment (earth2travis) [inline pages/getting-started.mdx:20]: Any reason we are removing these links?

File: pages/getting-started.mdx [hunk 2]
```diff
@@ -17,36 +17,66 @@ We're using a monorepo to manage our libraries and apps. Each library can be ins
...
+- DAOhaus Connect
+- Tx Builder
+- Form Builder
 
 Each of these libraries can be found in our [DAOhaus v3 Monorepo](https://github.com/HausDAO/daohaus-monorepo) in the `/libs` folder and are included in the Toolbox documentation.
 
 ## DAOhaus v3 Monorepo Overview
 
-Our monorepo follows the [recommended Nx structure](https://nx.dev/more-concepts/folder-structure#integrated-repo-folder-structure) with `apps` and `libs` folders. The `apps` folder contains our core applications, subgraphs, and infrastructure jobs. If you want to see how we utilize the libraries in the _DAOhaus Toolbox_ in our own v3 app development check out the packages in this folder.
+Our monorepo follows the [recommended Nx structure](https://nx.dev/structure/applications-and-libraries) with `apps` and `libs` folders. The `apps` folder contains our core applications, subgraphs, and infrastructure jobs. If you want to see how we utilize the libraries in the _DAOhaus Toolbox_ in our own v3 app development check out the packages in this folder.
```
Comment (earth2travis) [inline pages/getting-started.mdx:36]: https://nx.dev/structure/applications-and-libraries is broken

Comment (jonathanprozzi) [reply pages/getting-started.mdx:36]: I'll remove the link -- that must have been removed recently

**Review by earth2travis — APPROVED** [review]
👀 ✅

**Pull request merged**